### PR TITLE
1244853: Clean up product-details usage

### PR DESCRIPTION
**Pull request by willkg**
This adds product-details updates to the deploy and also nixes use of
the kuma/languages.json file.

To review:

1. make sure the approach is ok
2. run tests (Travis will do that)

We could do a test deploy on -stage, but I copied the deploy code from
Input so I'm pretty confident it's good. If there are issues, we can
just do a followup fix.

r?

### CODE REVIEW

**Comment by willkg** [comment]
The settings changes just landed. I'll rebase this shortly.

**Comment by willkg** [comment]
^^^ That fixes the merge conflict.

File: settings.py [hunk 1]
```diff
@@ -15,7 +15,1348 @@
     # when the code runs on the admin node during deploy
     IS_STAGE = CHIEF_REMOTE_HOSTNAME == 'developer.allizom.org'
 
+<<<<<<< 79a9d5637ab4fca2d785d624516dabcf950d6051
 if IS_STAGE:
     from kuma.settings.stage import *  # noqa
 else:
     from kuma.settings.prod import *  # noqa
+=======
```
Comment (robhudson) [inline settings.py]: Not pep8 compliant.

File: scripts/chief_deploy.py [hunk 2]
```diff
@@ -42,6 +42,13 @@ def update_code(ctx, tag):
 
 
 @task
+def update_product_details(ctx):
+    """Update mozilla product details files"""
+    with ctx.lcd(settings.SRC_DIR):
+        ctx.local("python2.7 manage.py update_product_details -f")
```
Comment (robhudson) [inline scripts/chief_deploy.py]: You added this but it's not called anywhere?

**Comment by willkg** [comment]
Ugh. Mistakes! Boo hiss!

File: settings.py (hunk 1 above)
Comment (willkg) [inline settings.py]: So screwy. I did a rebase, hit a merge conflict, fixed it just fine and its as if the rebase didn't finish or something weird.

**Comment by willkg** [comment]
^^^ That fixes the merge-confict stuff in `settings.py` and also adds a call to `update_product_details` in the deploy script.

File: scripts/chief_deploy.py (hunk 2 above)
Comment (robhudson) [inline scripts/chief_deploy.py]: Do we also want to add the `-f` (force) to vagrant provisioning?

**Comment by robhudson** [comment]
this looks good to me. I'm r+ with a simple question.

**Comment by willkg** [comment]
^^^ That adds the `-f` for provisioning. I tested it out with a clean vagrant install. I don't think it'll affect much and certainly won't affect travis tests. I'll merge this later.

**Comment by willkg** [comment]
Merging this in.

**Pull request merged**